sdk/bento/graph/__init__.py

```diff
@@ -38,10 +42,18 @@ def compile_graph(
 ) -> Graph:
     """Compiles the given `convert_fn` into a computation Graph.
 
-    Compiles by converting the given `convert_fn` function to AST,
+    Compiles by converting the given `convert_fn` function to AST
+    applying the given `preprocessors` transforms to perform preprocessing on the AST,
     applying given `analyzers` on the AST to perform static analysis,
     linting the AST with the given `linters` to perform static checks,
-    applying the given `transforms` to transform the AST.
+    applying the given `transforms` to transform the AST to a function that plots the computational graph.
+
+    Note:
+        Even though both `preprocessors` and `transforms` are comprised of  a list of `Transform`s
+        `preprocessors` transforms are applied before any static analysis is done while
+        `transforms` are applied after static analysis. This allows `preprocessors` to focus
```

Comment:
How about just calling it `postprocessors` instead of `transforms`?

Reply:
Because this aggressively transforms the AST instead of "postprocessing" it. For example post processing in a movie might mean adding special effects/editing, but the content is still there. `transforms` changes it completely.